setup/local-preview.md

```diff
@@ -0,0 +1,62 @@
+---
+title: "Local Preview"
+description: Set up a Coder deployment locally for testing.
+---
+
+Coder is typically deployed onto Kubernetes cluster, but if you would like to
+set up a lightweight preview deployment, you can do so locally using using
+[Docker](https://www.docker.com/) and [kind](https://kind.sigs.k8s.io/).
+
+> Coder currently supports local preview only on workstations running macOS or
+> Linux.
+
+## Prerequisites
+
+Before proceeding, please make sure that you have the following installed:
+
+1. [Docker](https://hub.docker.com/search?q=docker&type=edition&offering=community)
+2. [kind](https://kind.sigs.k8s.io/docs/user/quick-start/#installation)
+3. [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl)
+4. [helm](https://helm.sh/docs/intro/install)
+
+## Installing Coder
+
+To install Coder, run:
+
+```bash
+curl -fsSL https://coder.com/try.sh | PORT="8080" sh
```

Comment:
This script is not POSIX-compliant so has to be run using bash, not sh.
(alpine errors out syntax, wsl (debian) errors about pipefail)

Reply:
Thanks for the report. I'll switch this to bash for now and see if anyone has issues with the requirement of bash over sh.

Reply:
I mean... if script is not POSIX only - normal sh wouldn't be able to run this.

Reply:
Sorry, I meant going with bash vs removing the pipefail (or other potentially non-POSIX things) to stick with sh.

Reply:
This definitely emphasizes getting testing setup to make sure this works correctly in the standard bash shipped with most OSes. I'll definitely try and prioritize that before this gets much marketing push.

Reply:
Hey @Duelsik, pushed an update last night that made the script POSIX compatible and tested it on busybox (alpine) sh. It should now be fully POSIX now.